docs/proposals/20220725-managed-kubernetes.md

```diff
@@ -435,11 +442,9 @@ The reasons for this recommendation are as follows:
 - The infra cluster provisions and manages the general infrastructure required for the cluster but not the control plane.
 - By having a separate infra cluster API definition, it allows differences in the API between managed and unmanaged clusters.
 
-Providers like CAPZ and CAPA have already implemented managed Kubernetes support and there should be no requirement on them to move to Option 3. Both Options 2 and 4 are solutions that would work with ClusterClass and so could be used if required.
-
-Option 1 is the only option that will not work with ClusterClass and would require a change to CAPI. Therefore this option is not recommended.
+Providers like CAPZ and CAPA have already implemented managed Kubernetes support and there should be no requirement on them to move to Option 3. Both Options 4 also works well with ClusterClass and so could be used if required. Option 2 will no longer be relevant once CAEP https://github.com/kubernetes-sigs/cluster-api/pull/8500 is implemented.
 
-*** This means that CAPA will have to make changes to move away from Option 1 if it wants to support ClusterClass.
+Option 1 will be available when CAEP https://github.com/kubernetes-sigs/cluster-api/pull/8500 is implemented. Until then it is not recommended.
```

Comment:
This is not right as it stands with the title of option 1.

We should edit option 1 so its called: `Option 1: Single kind for Control Plane and no infra cluster` and then make whatever adjustments to the body of the section. 

We need to call out this is for "infrastructure less" managed k8s implementations.